operations/toc-chair.md

```diff
@@ -0,0 +1,40 @@
+# TOC Chair
+
+This document serves to provide the TOC Chair with details on their responsibilities, role, and expectations that are in addition to those of a TOC member.
+
+## Selecting the Chair
+
+The TOC Chair is an member of the TOC that is chosen by each seated TOC to lead and presented the TOC in the execution of it's work. 
+
+## Functions of the Chair
+
+The Chair is expected to perform a variety of functions and can often be thought of as a Team Lead, as such their work relies on the input and support of the TOC to be successful. A list of known functions and their details is below:
+
+* Facilitate Initiative Planning
+* Coordinate TOC activities
+* Ensure Liaisons are assigned and engaged for each TAG
+* Coordinate overarching TAG activities
+* Represent the TOC to the Governing Board
```

Comment:
may be expand a bit to say that they have a seat on the board (not just represent the TOC ...).

Reply:
I think adding a bullet around being a "technical public spokesperson on behalf of CNCF TOC" as the TOC Chair has been used a lot before in media interviews, quotes, etc - especially around kubecon time